website/source/layouts/aws.erb

```diff
@@ -10,6 +10,15 @@
                     <a href="/docs/providers/aws/index.html">AWS Provider</a>
                 </li>
 
+                <li<%= sidebar_current(/^docs-aws-datasource/) %>
```

Comment:
I think the AWS docs page will have to be redesigned - the more data sources we add, the longer that left hand nav is going to be :)

This is obviously a good thing though